syntax = "proto3";

package stock;

import "google/protobuf/timestamp.proto";

service StockService {
  rpc GetStock (GetStockRequest) returns (GetStockResponse);
}

message GetStockRequest {
  repeated int64 item_ids = 1;
}

message GetStockResponse {
  repeated StockItem items = 1;
}

message StockItem {
  int64 item_id = 1;
  int64 warehouse_id = 2;
  int32 quantity = 3;
  google.protobuf.Timestamp updated_at = 4;
} 